django/db/models/query.py

```diff
@@ -820,6 +837,7 @@ def bulk_create(
                     on_conflict=on_conflict,
                     update_fields=update_fields,
                     unique_fields=unique_fields,
+                    where_clause=where_clause,  # Ajouter where_clause ici
```

Comment:
Please don't add unnecessary comments saying what the code does. Additionally all comments are in US English. ☺️

Reply:
I understand, I'll remove any unnecessary comments. Thanks for the reminder about the comment language.